src/NodaTime/Calendars/HebrewScripturalCalculator.cs

```diff
@@ -211,7 +213,7 @@ private static bool IsKislevShort(int year)
         internal static int ElapsedDays(int year)
         {
             int cache = GetOrPopulateCache(year);
-            return cache & ElapsedDaysCacheMask;
+            return cache >> 2;
```

Comment:
Not sure if it's worth pulling out that `2` into an `ElapsedDaysShift` or similar?